crates/torii/libp2p/src/server/mod.rs

```diff
@@ -324,6 +309,8 @@ impl<P: Provider + Sync> Relay<P> {
                                 };
 
                             let mut calldata = vec![message_hash];
+                            calldata.push(Felt::TWO);
```

Comment:
Hum, not sure why this one is required. Because the signature is not always 2 felts long.
Would be better to prepend the actual length of `data.signature` instead of hard coded value.

@Larkooo should we expect the signature coming from the off-chain message to send a serialized vector already (hence containing the length), or only the buffer and we should prepend the length?

Reply:
Libp2p expects a vec of felts for signatures, so we should always assume that signatures are an arbitrary number of felts. So yes we should instead put the length of the signatures vec

Reply:
Ok makes sense, fixed in my latest commit